### PR TITLE
(slightly) rework either

### PR DESCRIPTION
**Pull request by JAicewizard**
This is a very simple example with a very simple change.
I am thinking about just removing al the localization from every example and make a special example with that, it is very nice someone translated all of them but a dedicated example probably works best. This could showcase things like localization arguments etc that are not really shown off in any of the current examples.

### CODE REVIEW

File: druid/examples/either.rs [hunk 1]
```diff
@@ -12,42 +12,43 @@
...
+        Label::new("Click to reveal slider").padding(5.0),
     );
-    col.add_child(either);
-    col
+    Flex::column().with_child(button).with_child(either)
+}
+
+pub fn main() {
+    let main_window =
+        WindowDesc::new(ui_builder).title(druid::widget::LabelText::from("Switcheroo"));
```
Comment (cmyr) [inline druid/examples/either.rs:49]: you should be able to just write `.title("Switcheroo")`.

File: druid/examples/either.rs [hunk 2]
```diff
@@ -12,42 +12,43 @@
...
+        .lens(AppState::which)
+        .padding(5.0);
 
-    let mut col = Flex::column();
-    col.add_child(
-        Checkbox::new("Toggle slider")
-            .lens(AppState::which)
-            .padding(5.0),
-    );
+    // The either widget has 2 children, one of which is visible.
```
Comment (cmyr) [inline druid/examples/either.rs:35]:
```suggestion
    // The Either widget has two children, one of which is visible at a time.
```

english convention varies, but a good guideline for technical writing is to write numbers under 10 (so up to nine, inclusive 😉) as words.

File: druid/examples/either.rs [hunk 3]
```diff
@@ -12,42 +12,43 @@
...
+    // you have to pass in a closure which takes `Data` and and `Env`.
+    // This closure determines which widget gets displayed based on the
+    // return value (a bool). False is the second widget and true is the first.
```
Comment (cmyr) [inline druid/examples/either.rs:38]:
```suggestion
    // The `Either` widget has two children, only one of which is visible at a time.
    // To determine which child is visible, you pass it a closure that takes the
    // `Data` and the `Env` and returns a bool; if it returns `true`, the first
    // widget will be visible, and if `false`, the second.
```

english convention varies, but a good guideline for technical writing is to write numbers under 10 (so up to nine, inclusive 😉) as words.

**Review by cmyr — COMMENTED** [review]
Few little things, but looks good!

File: druid/examples/either.rs (hunk 2 above)
Comment (JAicewizard) [reply druid/examples/either.rs:35]: Yeah it is dutch convention too for under 20, I usually forget about it.
If anything I prefer just numbers as numbers to keep consistency.

File: druid/examples/either.rs (hunk 3 above)
Comment (JAicewizard) [reply druid/examples/either.rs]: If my writing is very bad let me know, I can put more effort into it.
I don't have a preference here but maybe you view that completely different.

File: druid/examples/either.rs [hunk 4]
```diff
@@ -12,42 +12,43 @@
...
+        .padding(5.0);
 
-    let mut col = Flex::column();
-    col.add_child(
-        Checkbox::new("Toggle slider")
-            .lens(AppState::which)
-            .padding(5.0),
-    );
+    // The Either widget has two children, one of which is visible at a time.
+    // you have to pass in a closure which takes `Data` and and `Env`.
```
Comment (cmyr) [inline druid/examples/either.rs:36]:
```suggestion
    // you have to pass in a closure which takes `Data` and `Env`.
```

File: druid/examples/either.rs [hunk 5]
```diff
@@ -12,42 +12,43 @@
...
+        Label::new("Click to reveal slider").padding(5.0),
     );
-    col.add_child(either);
-    col
+    Flex::column().with_child(button).with_child(either)
+}
+
+pub fn main() {
+    let main_window =
+        WindowDesc::new(ui_builder).title(druid::widget::LabelText::from("Switcheroo"));
```
Comment (cmyr) [inline druid/examples/either.rs:49]:
```suggestion
        WindowDesc::new(ui_builder).title("Switcheroo");
```

File: druid/examples/either.rs (hunk 4 above)
Comment (JAicewizard) [reply druid/examples/either.rs:36]: :+1:
I am no longer behind my computer so I couldn't commit myself, thanks!

File: druid/examples/either.rs (hunk 5 above)
Comment (JAicewizard) [reply druid/examples/either.rs:49]: :+1:
I am no longer behind my computer so I couldn't commit myself, thanks!

**Review by cmyr — APPROVED** [review]
Thanks!

**Pull request merged**